lib/logstash/outputs/elasticsearch/http_client_builder.rb

```diff
@@ -197,5 +197,14 @@ def self.setup_api_key(logger, params)
     def self.dedup_slashes(url)
       url.gsub(/\/+/, "/")
     end
+
+    def self.resolve_filter_path(url)
+      return url if url.match?(/(?:[&|?])filter_path=/)
+      ("#{url}#{query_param_separator(url)}filter_path=errors,items.*.error,items.*.status")
```

Comment:
Can you add a note about this injection of filter_path and it's skipping in the docs?

Reply:
done